LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -940,12 +940,15 @@ TacticalText="<Bullet/> Coil weapons automatically pierce (ignore) 1 enemy armor
 AbilityDescName="sniper rifle"
 
 [Shotgun_CG X2WeaponTemplate]
-FriendlyName="Scattergun"
-FriendlyNamePlural="Scatterguns"
+; LWOTC needs translation
+FriendlyName="Gauss Scattergun"
+FriendlyNamePlural="Gauss Scatterguns"
 BriefSummary="A close-quarters coil-tech weapon that fills the shotgun role."
```

Comment:
`; End Translation` should be put above this line. You haven't changed it, nor the one below, so they don't need translating.

Reply:
So, I think I'm understanding how this is all going. Since I changed another line just below that one, I just open and close "translation" as needed.

2911afe84d133ba0b30049f310e0d945bc58a5c3